pkg/systemd/README-realmd.md

```diff
@@ -1,3 +1,5 @@
+!! THIS IS A BIT OUT OF DATE BUT MIGHT STILL BE USEFUL !!
```

Comment:
Should actually be okay except that the image is now "services", and one needs to call /run-freeipa like test/verify/check-realms does.

Reply:
Fixed